doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/09_Relation_Types.md

```diff
@@ -96,6 +96,9 @@ $relationObject = $relation->getObject();
 DataObject\AbstractObject::setHideUnpublished(true);
 ```
 
+### Many-To-Many Object Relation
+If enabled in the class definition field settings a filter will be displayed. The filter currently supports filtering texts, numbers and links.
```

Comment:
this comment in docs needs to be removed again ;-)

Reply:
thanks, done!